airflow/www/templates/airflow/trigger.html

```diff
@@ -47,22 +129,90 @@ <h2>Trigger DAG: {{ dag_id }}</h2>
         <input type="text" class="form-control" placeholder="Run ID" name="run_id">
       </div>
     </div>
+    {% if recent_confs|length > 0 %}
     <div class="form-group row">
       <div class="col-md-2">
-        <label for="recent_configs">Recent Configurations</label>
+        <label for="recent_configs">Select Recent Configurations</label>
         <select class="form-control" name="recent_configs" id="recent_configs">
-          <option value="">empty config</option>
-          {% for conf in recent_confs %}
-          <option value="{{ conf }}">{{ conf }}</option>
+          <option value="{{ conf }}">Default parameters</option>
+          {% for run_id, recent_conf in recent_confs.items() %}
+          <option value="{{ recent_conf }}">{{ run_id }}: {{ recent_conf }}</option>
           {% endfor %}
```

Comment:
This section is the select of the recent configs which can be applied to the form:
![image](https://user-images.githubusercontent.com/95105677/214951905-5b12c900-106f-4f71-91b6-941401d2746f.png)